nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pipes/component-type-name.pipe.ts

```diff
@@ -23,7 +23,10 @@ import { ComponentType } from '../state/shared';
     standalone: true
 })
 export class ComponentTypeNamePipe implements PipeTransform {
-    transform(type: ComponentType): string {
+    transform(type: ComponentType | string): string {
+        if (typeof type === 'string') {
+            return type;
+        }
```

Comment:
this is an issue too, sorry for the bad suggestions. we need to remove this check and just return `type` in the default case of the switch...